Tasks/Common/RemoteDeployer/JobHelper.ps1

```diff
@@ -0,0 +1,189 @@
+$defaultScriptArgumentsValues = @{
+    scriptPath = "";
+    scriptArguments = "";
+    inlineScript = "";
+    inline = $true;
+    workingDirectory = "";
+    errorActionPreference = "continue";
+    ignoreLASTEXITCODE = $false;
+    failOnStdErr = $false;
+    initializationScriptPath = "";
+    sessionVariables = "";
+}
+
+function Run-RemoteScriptJobs {
+    [CmdletBinding()]
+    Param (
+        [System.Management.Automation.Runspaces.PSSession[]] $sessions,
+        [scriptblock] $script,
+        [string] $sessionName,
+        [hashtable] $scriptArgumentsByName,
+        [hashtable[]] $targetMachines,
+        [scriptblock] $outputHandler,
+        [scriptblock] $errorHandler
+    )
+    Trace-VstsEnteringInvocation -InvocationInfo $MyInvocation -Parameter ""
```

Comment:
add write-verbose log for machine names

Reply:
done. logged it in the place where we are creating each of the target machines object